server/lib/geolocation.js

```diff
@@ -0,0 +1,13 @@
+const request = require('superagent');
+
+const getGeoLocation = (postcode, callback) => {
+  request
+   .get(`https://api.postcodes.io/postcodes/${postcode}`)
+   .end((err, res) => {
+     callback(err, res.body.result);
+   });
+};
+
+module.exports = {
```

Comment:
Is this module likely to export anything other than getGeoLocation?  If not then does it make sense just to export the function itself, rather than an object?